frontend/ui/translator.lua

```diff
@@ -540,12 +579,14 @@ function Translator:_showTranslation(text, target_lang, source_lang, from_highli
             table.insert(source, s)
             table.insert(translated, t)
         end
-        table.insert(output, "▣ " .. table.concat(source, " "))
-        text_main = "● " .. table.concat(translated, " ")
+        if not full_page then
+            table.insert(output, "▣ " .. table.concat(source, " "))
+        end
+        text_main = (full_page and "" or "● ") .. table.concat(translated, " ")
```

Comment:
(When naming this new parameter "full_page", did you think about translating the current full page - or showing the result translation full page ? I would have named it "no_alt_translation" or "simple_translation" as you're just ignoring all the other translations/definitions. But may be this ambivalent "full_page", because it is ambivalent, is fine.)

Reply:
I think this snippet would read better if you kept the 2 ways separate, rather than `(full_page and "" or "● ") ..`:
```lua
if full_page then
        text_main = table.concat(translated, " ")
else
        table.insert(output, "▣ " .. table.concat(source, " "))
        text_main = "● " .. table.concat(translated, " ")
end
```